src/buildercore/terraform.py

```diff
@@ -10,6 +10,14 @@
 RESOURCE_TYPE_FASTLY = 'fastly_service_v1'
 RESOURCE_NAME_FASTLY = 'fastly-cdn'
 
+FASTLY_GZIP_TYPES = ['text/html', 'application/x-javascript', 'text/css', 'application/javascript',
+                     'text/javascript', 'application/json', 'application/vnd.ms-fontobject',
+                     'application/x-font-opentype', 'application/x-font-truetype',
+                     'application/x-font-ttf', 'application/xml', 'font/eot', 'font/opentype',
+                     'font/otf', 'image/svg+xml', 'image/vnd.microsoft.icon', 'text/plain',
+                     'text/xml']
+FASTLY_GZIP_EXTENSIONS = ['css', 'js', 'html', 'eot', 'ico', 'otf', 'ttf', 'json']
```

Comment:
is there a standard way to sort these, or should we do it in the code, so that the templates stays stable?

Reply:
This is the order on https://docs.fastly.com/guides/basic-configuration/enabling-automatic-gzipping.

Reply:
We could sort in https://github.com/elifesciences/builder/pull/287/files#diff-c266af1366f88d308f6fcb68d77552bcR46 so that doing a diff in the future is easier?

Reply:
`sorted()`